app/controllers/accounts/contacts/events_controller.rb

```diff
@@ -33,14 +38,25 @@ def create
       return render :new, status: :unprocessable_entity
     end
   end
+  def destroy
+    @event.destroy
+    render turbo_stream: [
+      turbo_stream.remove(@event)
+    ]
```

Comment:
Esse trecho aqui talvez seja melhor colocar no model, assim se fizermos uma deleção por exemplo pelo sidekiq ou até mesmo console o elemento é removido da página.
Segue um exemplo:
https://youtu.be/xSkq9vQgT7k?si=-fY_L2QTrvmXxU5c&t=534